src/components/OcSelect.vue

```diff
@@ -187,4 +286,9 @@ export default {
 }
 </style>
 ```
+
+## Loading spinner
+```js
+<oc-select :options="['Apple', 'Bannana', 'Orange', 'Pear']" :multiple="true" :loading="true" />
```

Comment:
This one breaks the dropdown-preview for me. Could you take a look? :)

Reply:
Dropdown shouldn't be visible at all since it is in loading state (clarified in chat)